qiita_pet/handlers/base_handlers.py

```diff
@@ -25,10 +25,13 @@ def write_error(self, status_code, **kwargs):
             self.render("404.html", user=self.current_user)
             return
 
+        is_admin = False
         if self.current_user:
-            is_admin = User(self.current_user).level == 'admin'
-        else:
-            is_admin = False
+            try:
+                is_admin = User(self.current_user).level == 'admin'
+            except:
```

Comment:
It was closed outside of this thread. Essentially it's needed only because you guys wanted a link to the error page for admins, so this bulletproofs that,